NativeYoutube/Models/Constants.swift

```diff
@@ -30,6 +30,8 @@ enum StatusStates: String {
 enum AppStorageStrings: String {
     case apiKey = "Api Key"
     case playListID = "Playlist ID for your goto playlist"
+    case videoClickBehaviour = "Behaviour when double click on a video"
```

Comment:
Please use.

"com.aayush.nativeyoutube.videoClickBehaviour"

I'm planning on migrating all AppStorageStrings as the naming scheme is all over the place

Reply:
Okay ! So basically change the value of the videoClickBehaviour enum case to "com.aayush.nativeyoutube.videoClickBehaviour" i suppose ?

Reply:
yes!, I'll push my migration code which renames all those Strings to "com...." schemes.

if we rename all of them now they'll lose persistence, causing users to get the API KEYs again